submissions/jeffersonMormah/javaScipt2/script.js

```diff
@@ -0,0 +1,18 @@
+var a = 1
```

Comment:
This declaration is not needed @Jeffbuchi

Reply:
Removed the first declaration